File: src/P2_18/P2_18.scala
object P2_18 {
    def getDiscountPercentage(items: List[String]): Int = {
        if(items.contains("Book")){
            5
        }else {
            0
        }
    }
}

@main def runScript(): Unit = {
  println(P2_18.getDiscountPercentage(List("Apple")) == 0);
  println(P2_18.getDiscountPercentage(List("Book")) == 5);
}

File: src/P2_18/P2_18.java
package P2_18;

import java.util.List;

public class P2_18 {
    public static void main(String[] args) {
        System.out.println(getDiscountPercentage(List.of("Apple")) == 0);
        System.out.println(getDiscountPercentage(List.of("Book")) == 5);
    }

    public static int getDiscountPercentage(List<String> items) {
        if (items.contains("Book")) {
            return 5;
        } else {
            return 0;
        }
    }
}
